src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Security/Services/IPermissionsService.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using OrchardCore.Security.Permissions;
+using OrchardCore.Environment.Extensions;
+
+namespace OrchardCore.Security.Services
+{
+    /// <summary>
+    /// Service that returns the installed permissions
+    /// </summary>
+    public interface IPermissionsService
+    {
+        Task<IEnumerable<Permission>> GetInstalledPermissionsAsync();
+    }
+
+    public class BasePermissionsService : IPermissionsService
```

Comment:
```suggestion
    public class DefaultPermissionsService : IPermissionsService
```

Reply:
Also this class shouldn't be in the abstractions project.

The interface should, but the implementation should be in the `Infrastructure` project